ivanti_security_controls/icon_ivanti_security_controls/actions/start_patch_scan/action.py

```diff
@@ -3,8 +3,7 @@
 
 # Custom imports below
 from insightconnect_plugin_runtime.exceptions import PluginException
-import time
-
+import polling2
```

Comment:
Open question for @mrinehart-r7 @jschipp-r7, do we want to include little libraries for polling, etc?  Seems like another dependency to manage/update but I get that we use it often.

Reply:
This one is fine with me 👍